django_project/minisass_frontend/src/components/AuthenticationButtons/index.tsx

```diff
@@ -88,13 +88,12 @@ function AuthenticationButtons() {
       });
   
       if (response.status === 201) {
-        setError(false)
         setLoading(true)
         // Simulate 2-second delay for registration process
         setTimeout(() => {
           setLoading(false);
           setRegistrationInProgress(true);
-        }, 1100);
+        }, 1200);
```

Comment:
@tinashechiraya  Why do we need to simulate this in the frontend?

Reply:
@dimasciput it's more of  just a styling thing where the user submits their details and it just shows the loading bar for a second then it shows the message with the instructions on how to complete the registration